### ISSUE TITLE
This is about the Max distance and the update delay sliders

### ISSUE THREAD

**Issue by mjoe67886** [semagnum/nview-live#13]
I think it would be a more professioal add on if it had those little buttons to the right and left of the Max distance.. Cause I'm a blender power user I move fast and I I like things to be secure and precise so I could just move my mouse around the viewport and get things done if I could just reach up there and and know I could set the setting quickly and precisely every time that would be an awesome bonus... Because now cause you're add on I'm I'm working on editing big cities in blender and doing animations for a sitcom I'm creating in blender in those cities and if I could just you know rock out in material tab and edit the materials of the city real quick like you're add on would allow me to do instead of blender freezing every 5 seconds that would be awesome... And just know I'm working on a tutorial seriescalled creating a sitcom in blender right now that's gonna highlight all my awesome speedy workflow tips and uh you're add  on is one that I'm going to really promote the crap out of because it's genius... Just so you know I'm an efficiency expert I'm always looking for ways to save button clicks in blender... Because the more content you can produce the bigger you could get on YouTube

**Comment by semagnum** [comment]
Thank you for mentioning this! I'll add these changes in for the distance and delay properties. I should be able do it sometime tomorrow - if not, then over the weekend.

**Comment by mjoe67886** [comment]
😆Yay I made a difference in the blender community yeah I'm going to really
promote this add on this is genius ....you should suggest to right click
select that they make this a part of blender

**Comment by mjoe67886** [comment]
Just an addendum I had no idea why I wrote removing in the issue request just OMIT that it's about adding the little buttons

**Comment by mjoe67886** [comment]
Anybody that reads this after you install the update you have to restart blender to see the little buttons but other than that thank you very much for adding that... now when I get in real close I'm trying to fine tune my experience with the add on those little buttons will help me... I had issue with that and now that's fixed

**Comment by semagnum** [comment]
> You have to restart Blender to see the little buttons

Yes, this is the same with all add-ons: Blender caches add-ons while active, so you have to close Blender to refresh it. I mention this in the documentation as well: https://semagnum.github.io/nview-live/install
